packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.controller.js

```diff
@@ -42,14 +43,35 @@ export default class KubernetesNodePoolsScaleCtrl {
           this.$translate.instant('kube_node_pool_autoscaling_scale_success'),
         ),
       )
-      .catch((error) =>
-        this.goBack(
-          this.$translate.instant('kube_node_pool_autoscaling_scale_error', {
-            message: get(error, 'data.message'),
-          }),
-          'error',
-        ),
-      );
+      .catch((error) => {
+        if (get(error, 'data.status') === 412) {
+          this.goBack({
+            textHtml: this.$translate.instant(
+              `kube_node_pool_autoscaling_scale_error_${get(
+                error,
+                'data.message',
+              ).slice(
+                get(error, 'data.message').indexOf('[') + 1,
+                get(error, 'data.message').indexOf(']'),
+              )}`,
+              {
+                quotaUrl: this.coreURLBuilder.buildURL(
+                  'public-cloud',
+                  `#/pci/projects/${this.projectId}/quota`,
+                ),
+              },
+            ),
+            type: 'error',
+          });
```

Comment:
I think it's more readable like this.

```suggestion
          const errorMessage = get(error, 'data.message');
          const errorId = errorMessage.slice(errorMessage.indexOf('[') + 1, errorMessage.indexOf(']') );
          
          this.goBack({
            textHtml: this.$translate.instant(
              `kube_node_pool_autoscaling_scale_error_${errorId}`,
              {
                quotaUrl: this.coreURLBuilder.buildURL(
                  'public-cloud',
                  `#/pci/projects/${this.projectId}/quota`,
                ),
              },
            ),
            type: 'error',
          });
```